### DISCUSSION TITLE
Shorter read length

### DISCUSSION THREAD

**Discussion by robbueck** [bhattlab/phanta#35] [Phanta DBs] [answered]
Hi,
I'm working on a dataset where the reads are around 60 bp long after trimming. would it be possible to have a database that allows read_length to be 60 bp?
Thank you in advance
Rob

**Comment by meenachakra** [comment, marked as answer]
Hello, yes! You can create this yourself by activating your Phanta environment and executing the bracken-build command here, with read length 60 (default value for k) -
https://github.com/jenniferlu717/Bracken#step-1-generate-the-bracken-database-file-databasexmerskmer_distrib

let us know if you have any questions!

**Comment by robbueck** [comment]
I've tried that, but I'm not sure what I should give as database location. If I give the folder with the phanta database (unmasked_db_v1) it creates empty files. From what I get from the bracken docs, it seems that it misses a fasta file in the library dir?

**Comment by meenachakra** [comment]
Ah, I see, thanks for letting us know!
You can find the fna files here:
wget http://ab_phanta.os.scg.stanford.edu/Phanta_DBs/database_V1.library.tar.gz

You can download that into the database folder, untar it and rename it library if necessary... let us know if that works!

**Comment by zhixuanyan** [comment]
Hello, I have downloaded the updated database (HumGut + UHGV "HQ+"), but the read length of the bracken database in this database is 150bp. I want to build a bracken database with a read length of 100 bp, but I don't have the  genome file for ' HumGut + UHGV "HQ+" ' databse , can you tell me where to download it? Very much looking forward to your reply, thank you.

**Comment by meenachakra** [comment]
Sure! Will update soon.

**Comment by meenachakra** [comment]
Here you go! You can download this directly. Let us know if it doesn't work for you.
[100mers_humgut_uhgv_hqplus.zip](https://github.com/bhattlab/phanta/files/13631542/100mers_humgut_uhgv_hqplus.zip)

**Reply by zhixuanyan** [reply]
Thank you so much. This file is useful and I have tested successfully using it. But please forgive me for not making it clear last time. Because I also have a batch of 75bp sequences, so for the database 'HumGut + UHGV "HQ+"', I also need a 75bp bracken database.

**Reply by meenachakra** [reply]
Glad it's useful! Got it, we'll upload the entire library and let you know when that's done.

**Reply by meenachakra** [reply]
We're currently having some technical difficulties uploading to the repository that we normally upload to, but hopefully this will be resolved shortly. If not, we will share the 75bp Bracken DB with you.

**Reply by zhixuanyan** [reply]
OK, thanks.

**Reply by meenachakra** [reply]
We're unfortunately not able to figure it out - but here is the 75bp file in the meantime!
[75mers_kmer_distrib.zip](https://github.com/bhattlab/phanta/files/13680036/75mers_kmer_distrib.zip)

**Reply by zhixuanyan** [reply]
Thank you so much.

**Reply by meenachakra** [reply]
We've resolved the issue with the object store - and you can now download all the sequences using:

`wget http://ab_phanta.os.scg.stanford.edu/Phanta_DBs/humgut_uhgv_hqplus_library.tar.gz`
